lib/web_ui/lib/src/engine/skwasm/skwasm_impl/renderer.dart

```diff
@@ -401,18 +400,25 @@ class SkwasmRenderer implements Renderer {
   // TODO(harryterkelsen): Add multiview support,
   // https://github.com/flutter/flutter/issues/137073.
   @override
-  Future<void> renderScene(ui.Scene scene, ui.FlutterView view) =>
-      sceneView.renderScene(scene as EngineScene);
+  Future<void> renderScene(ui.Scene scene, ui.FlutterView view) {
+    view as EngineFlutterView;
+    assert(view is EngineFlutterWindow, 'Skwasm does not support multi-view mode yet');
```

Comment:
These asserts are removed in production (in fact I'm not sure skwasm has a concept of "debug" mode), what happens when we use multiview in skwasm? is it a runtime error?